### PR TITLE
timeline under about in navbnar

### PR DESCRIPTION
**Pull request by samumantha**
Roadmap or Timeline?

related #309

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -74,6 +74,7 @@
               <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                 <a class="dropdown-item" href="{{ config.base_url | safe }}/association/">Governance</a>
                 <a class="dropdown-item" href="{{ config.base_url | safe }}/about/code-of-conduct/">Code of Conduct</a>
+                <a class="dropdown-item" href="{{ config.base_url | safe }}/other/timeline/">Timeline</a>
```
Comment (bast) [inline templates/base.html:77]: why "other"?

**Comment by samumantha** [comment]
Maybe it would be good to do a little cleanup with those folders in general, I was not sure where things should be and things seem to be linked from random places. Would it be a good idea to use the navbar main elements like about, community etc as directory names to find things more easily?

**Comment by bast** [comment]
Yes that would be very good. It's nice when the main folders on the web map to folders in the sources.

**Comment by samumantha** [comment]
Should we merge this as is and create new issue to clean up the structure?

**Comment by lucaferranti** [comment]
> Should we merge this as is and create new issue to clean up the structure?

I think it would be a good idea

**Comment by lucaferranti** [comment]
opened #315 for this

**Pull request merged**